pkg/portal/portal.go

```diff
@@ -136,12 +137,22 @@ func (p *portal) setupRouter() error {
 	r := mux.NewRouter()
 	r.Use(middleware.Panic(p.log))
 
-	asset, err := Asset("index.html")
+	assetv1, err := Asset("v1/dist/index.html")
 	if err != nil {
 		return err
 	}
 
-	p.t, err = template.New("index.html").Parse(string(asset))
+	assetv2, err := Asset("v2/build/index.html")
+	if err != nil {
+		return err
+	}
+
+	p.tempv1, err = template.New("index.html").Parse(string(assetv1))
+	if err != nil {
+		return err
+	}
+
+	p.tempv2, err = template.New("index2.html").Parse(string(assetv2))
```

Comment:
`index.*` is a magic file in which web servers look for that file with particular extensions (html, php, js, etc...) and load the first file they find according to the web servers config.

since `index2.html` is not a magic/ reserved filename, should we rename this file to have more context? whats the difference between index.html and index2.html? 

In browser `index.html` won't be displayed (e.g google.com/index.html is equivalent to google.com),  but `index2.html` would. That doesn't feel like the best option. 

naming things is hard.

Reply:
I think initially I was having issues with using the admin portal index.html and I tried this and never tried to switch it back. So let me try and use index.html again and see if it works

Reply:
Reverted back to index.html and it works fine